packages/forma-36-website/src/pages/guidelines/date-and-time/index.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: 'Date and time'
+---
+
+import {
+  Table,
+  TableHead,
+  TableBody,
+  TableCell,
+  TableRow,
+  Tooltip,
+} from '@contentful/forma-36-react-components';
+
+# Date and time
+
+## Absolute vs relative
+
+Consider the context when having to choose between absolute and relative date and time.
+
+<Table className="f36-margin-bottom--xl" style={{ width: '100%' }}>
+  <TableHead>
+    <TableRow>
+      <TableCell style={{ width: '25%' }}>Type</TableCell>
+      <TableCell style={{ width: '25%' }}>Example</TableCell>
+      <TableCell style={{ width: '50%' }}>Notes</TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Absolute</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>13 Aug 2019 at 10:00 am</TableCell>
+      <TableCell>
+        When is an entry scheduled to be published?
+        <br />
+        Relative time wouldn't be specific enough.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Relative</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        <Tooltip content="13 Aug 2019 at 10:00 am">3 days ago</Tooltip>
+      </TableCell>
+      <TableCell>
+        When was my collaborator last active?
+        <br />
+        Absolute date and time would be unnecessarily specific. Provide the
+        absolute date and time on user interaction, e.g. when the user hovers
+        over relative time with their cursor.
+      </TableCell>
+    </TableRow>
+  </TableBody>
+</Table>
+
+## Formats
+
+<Table className="f36-margin-bottom--xl" style={{ width: '100%' }}>
+  <TableHead>
+    <TableRow>
+      <TableCell style={{ width: '25%' }}>Type</TableCell>
+      <TableCell style={{ width: '25%' }}>Example</TableCell>
+      <TableCell style={{ width: '50%' }}>Notes</TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Relative date and time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        Never
+        <br />
+        Less than an hour ago
+        <br />
+        13 days ago
+        <br />
+        3 months ago
+        <br />a year ago
+      </TableCell>
+      <TableCell></TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Absolute time</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>8:00 am</TableCell>
+      <TableCell>
+        Time should be displayed in 12-hour format, with lowercase am or pm and
```

Comment:
To make the text more easily scannable, time should be displayed in a 12-hour format. Use lowercase am or pm without periods.

Reply:
Making the text more easily scannable is not the reason behind the 12-hour format decision. It was an arbitrary decision.

Using lowercase am and pm, on the other hand, is because it's faster to recognize the difference between the two, as opposed to having them in UPPERCASE.

Therefore, I would rather omit the "To make it more easily scannable" for this case.